tools/loader/ImageClassifier.cpp

```diff
@@ -389,5 +416,5 @@ int main(int argc, char **argv) {
     loader.generateAndSerializeQuantizationInfos(bindings);
   }
 
-  return 0;
+  return numErrors;
```

Comment:
Specifying the correct index is optional, but if it's not specified then the program will exit with a non-zero return (assuming the input is not classified as `0`). If we move `expectedCategoryIndex` to a list then perhaps we can only return `numErrors` if the list is empty.

Reply:
Fixed in the last commit